test/test.js

```diff
@@ -189,6 +189,54 @@ describe('prompt()', function () {
 
         sendLine('yeaa');
     });
+
+    it('should prompt the user (using promise)', function (next) {
```

Comment:
Same thing for the tests.. I would only add one test per function, that actually test the promise usage. Could you change that?